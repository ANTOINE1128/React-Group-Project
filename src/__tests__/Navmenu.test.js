import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navmenu from '../components/Navmenu';

test('renders the navbar component snapshot', () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Navmenu />
    </BrowserRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
