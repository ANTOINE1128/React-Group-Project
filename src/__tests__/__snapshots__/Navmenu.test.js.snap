// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the navbar component snapshot 1`] = `
<DocumentFragment>
  <header>
    <nav
      class="container d-flex-space"
    >
      <div
        class="d-flex-space g-10"
      >
        <img
          alt="logo"
          class="logo-img"
          src="logo.png"
        />
        <h2
          class="logo-title"
        >
          Space Traveler's Hub
        </h2>
      </div>
      <ul
        class="d-flex-space g-10"
      >
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            class=""
            href="/missions"
          >
            Missions
          </a>
        </li>
        |
        <li>
          <a
            class=""
            href="/profile"
          >
            My Profile
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
